rust/routee-compass-powertrain/src/routee/prediction/interpolation/interpolation_speed_grade_model.rs

```diff
@@ -63,36 +66,41 @@ impl InterpolationSpeedGradeModel {
         grade_bins: usize,
         energy_rate_unit: EnergyRateUnit,
     ) -> Result<Self, TraversalModelError> {
-        // Load random forest binary file
-        let rf_binary = std::fs::read(routee_model_path).map_err(|e| {
-            TraversalModelError::FileReadError(
-                routee_model_path.as_ref().to_path_buf(),
-                e.to_string(),
-            )
-        })?;
-        // Load the random forest regressor so we can exercise it on a linear grid
-        let rf: RandomForestRegressor<f64, f64, DenseMatrix<f64>, Vec<f64>> =
-            bincode::deserialize(&rf_binary).map_err(|e| {
-                TraversalModelError::FileReadError(
-                    routee_model_path.as_ref().to_path_buf(),
-                    e.to_string(),
-                )
-            })?;
+        // load underlying model to build the interpolation grid
+        let model = load_prediction_model(
+            "".to_string(),
```

Comment:
this dummy string feels weird. the reason this is ok is because we actually aren't using this anywhere. it's a nice convention to include a name in the yaml though. something to ponder, whether we can make use of the name or if we should just ignore it.

Reply:
yeah I ended up passing the name down into the builder for the interpolation model since it was only up one layer and makes more sense than putting an empty string there.